src/meta/src/barrier/progress.rs

```diff
@@ -153,6 +153,8 @@ impl Progress {
 ///    On recovery, the stream manager will stop managing the job.
 /// 2. `Recovered`. This refers to the "Recovered" type of tracking job.
 ///    On recovery, the barrier manager will recover and start managing the job.
+/// 3. `Immediate`. Immediate jobs will immediately return,
+///    but still register the lag between the job and its upstream.
```

Comment:
Didn't find this variant? 👀

Reply:
Removed in https://github.com/risingwavelabs/risingwave/pull/13735/files, thanks!

It was used previously to indicate a new variant of stream job handling, but I realize I can just use a TrackingCommand without notifiers to accomplish the same thing, with less refactoring.